WordPress/Classes/ViewRelated/Post/PostSettingsViewController+JetpackSocial.swift

```diff
@@ -36,26 +36,31 @@ extension PostSettingsViewController {
         let isJetpackSocialEnabled = FeatureFlag.jetpackSocial.enabled
         let blogSupportsPublicize = apost.blog.supportsPublicize()
         let blogHasConnections = publicizeConnections.count > 0
-        // TODO: Check if there's a share limit
+        let isSocialSharingLimited = apost.blog.isSocialSharingLimited
+        let blogHasPublicizeInfo = apost.blog.publicizeInfo != nil
```

Comment:
An optional suggestion, but this one-liner should be enough to replace these; although your changes are also good to me. Feel free to decide!

```suggestion
        let blogHasSharingLimit = apost.blog.sharingLimit != nil
```

Reply:
Good call! Updated it with https://github.com/wordpress-mobile/WordPress-iOS/pull/21027/commits/769748545b2d84aa51debbcf1599e1277f771e3e.